src/com/musala/xml/model/Student.java

```diff
@@ -0,0 +1,49 @@
+package com.musala.xml.model;
+
+import javax.xml.bind.annotation.XmlAttribute;
+import javax.xml.bind.annotation.XmlElement;
+
+public class Student {
+
+    private String name;
+    private int age;
+    private String referenceNumber;
+    private int facultyNumber = 0;
```

Comment:
Does it make sense to initialize this variable with 0? Is it intended to be incremented (if students are to be added, it would, but currently, we just map it with the number from the XML).